src/interactableComponents/drawer/newDrawer.js

```diff
@@ -53,6 +53,10 @@ export default class NewDrawer extends BaseComponent {
      * Whether to give the items equal width (the max width)
      */
     equalWidths: PropTypes.bool,
+    /**
+     * Set a different minimum width
+     */
+    minWidth: PropTypes.number,
```

Comment:
I would change to `itemsMinWidth` to keep it similar to other props (`itemsTintColor`, `itemsIconSize`, etc..)

Reply:
Do we want all items to have the same minimum width? - Or maybe add item.minWidth?

Reply:
Done